### PR TITLE
Improved popup position

### PR DESCRIPTION
**Pull request by thinkh**
Related issue #340.

New dependencies: [popper.js](https://popper.js.org)

Merges into header-redesign branch.

### CODE REVIEW

File: src/dialogs/ADialog.ts [hunk 1]
```diff
@@ -26,13 +26,25 @@ abstract class ADialog {
...
+  protected static registerPopup(popup: HTMLElement, popper: Popper, replace: boolean) {
     if(replace) {
       ADialog.removeAllPopups();
     }
     if(ADialog.visiblePopups.length === 0) {
       popup.ownerDocument.addEventListener('keyup', escKeyListener);
     }
+
+    const closePopupOnMouseLeave = () => {
+      if(ADialog.visiblePopups.indexOf(popup) < ADialog.visiblePopups.length - 1) {
```
Comment (sgratzl) [inline src/dialogs/ADialog.ts]: not the last one?

Comment (sgratzl) [reply src/dialogs/ADialog.ts]: `ADialog.visiblePopups[ADialog.visiblePopups.length - 1] !== popup`

File: src/styles/_dialogs.scss [hunk 2]
```diff
@@ -25,6 +25,16 @@
     font-size: $lu_dialog_font_size;
     clear: right;
 
+    &::before {
```
Comment (sgratzl) [inline src/styles/_dialogs.scss]: do give it more space for mouse leave, right?

Comment (thinkh) [reply src/styles/_dialogs.scss]: Yes, exactly. So that the menu doesn't close already at the border.

File: src/dialogs/ADialog.ts (hunk 1 above)
Comment (thinkh) [reply src/dialogs/ADialog.ts]: Yes, much clearer ;-)

**Comment by thinkh** [comment]
@sgratzl Please check again or merge.

**Pull request merged**